groups/constants.js

```diff
@@ -1,5 +1,14 @@
 const NO_SPACES_ALLOWED = 'Roles cannot have spaces';
 const CANNOT_CONTAIN_GROUP = "Roles cannot contain 'group'.";
 const DEV_FEATURE_FLAG = 'dev';
+const SortByFields = [
+  { fieldName: 'memberCount', id: '1' },
+  { fieldName: 'date._seconds', id: '2' },
```

Comment:
NIT: having id declared first is more readable.

Reply:
Sure, done